### ISSUE TITLE
ScalaCheck properties don't fail

### ISSUE THREAD

**Issue by tomjadams** [etorreborre/specs2#581]
Checking ScalaCheck properties using `properties` vs. not results in different behaviour:

```scala
s2"Hello checks - should fail and does ${properties(prop)}"
```

vs.

```scala
s2"Hello checks - should fail and does not $prop"
```

Results in successful tests where they should fail.

See: https://github.com/tomjadams/specs2-scalacheck-bug/blob/master/src/test/scala/example/HelloSpec.scala for a fully worked & runnable example.

Reading the user guide, I'd expect this to work, and we have about a year's worth of tests written in this style that seem to no longer work (i.e. no longer fail when they should).

The output is:

```shell
> test
[info] HelloSpec
[info] 
[info] Hello checks - should fail and does 
[error] x Hello.greetings
[error]  Falsified after 0 passed tests.
[error]  > ARG_0: ""
[error]  > '' != 'this should fail!' (HelloSpec.scala:11)
[info] 
[info] + Hello checks - should fail and does not
[info] 
[info] 
[info] Total for specification HelloSpec
[info] Finished in 296 ms
[info] 2 examples, 1 expectation, 1 failure, 0 error
[info] 
[error] Failed: Total 2, Failed 1, Errors 0, Passed 1
[error] Failed tests:
[error] 	example.HelloSpec
[error] (test:test) sbt.TestsFailedException: Tests unsuccessful
[error] Total time: 2 s, completed 22/06/2017 1:27:13 PM
```

**Comment by tomjadams** [comment]
Reading the user guide, I'd expect this to work, and we have about a year's worth of tests written in this style that seem to no longer work (i.e. no longer fail when they should).

**Comment by etorreborre** [comment]
Thanks for the reproducible case. A fix is on the way in `3.9.1-20170622122021-deb1f60`.

**Comment by tomjadams** [comment]
Awesome! Thanks @etorreborre!
